tests/CMakeLists.txt

```diff
@@ -37,10 +37,16 @@ if(LIBCOSIM_WITH_PROXYFMU)
             "proxyfmu_integration_unittest"
             "proxyfmu_library_unittest"
             )
+    list(APPEND tests
+            "proxyfmu_osp_config_parser_test"
+            )
 endif()
 
 include("AddTestExecutable")
 foreach(test IN LISTS tests)
+    if (LIBCOSIM_WITH_PROXYFMU)
+        list(APPEND dependencies "proxyfmu::proxyfmu-client")
```

Comment:
This line does nothing as `dependencies` is not used.

Reply:
Removed in latest commit